packages/components/src/renderers/DefaultRenderer.spec.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { fromJS } from 'immutable';
+import renderer from 'react-test-renderer';
+
+import { DefaultRenderer } from '..';
+
+describe('DefaultRenderer', () => {
+    test('undefined', () => {
+        const component = <DefaultRenderer data={undefined} />;
+        const tree = renderer.create(component).toJSON();
+        expect(tree).toMatchSnapshot();
+    });
+
+    test('string', () => {
+        const component = <DefaultRenderer data={'test string'} />;
+        const tree = renderer.create(component).toJSON();
+        expect(tree).toMatchSnapshot();
+    });
+
+    test('boolean', () => {
+        const component = <DefaultRenderer data={true} />;
+        const tree = renderer.create(component).toJSON();
+        expect(tree).toMatchSnapshot();
+    });
+
+    // TODO MultiValueRenderer
```

Comment:
Is this meant to be done in this branch?

Reply:
Nope. I added test cases for the other scenarios of this component, but didn't want to mess with this at this time.